.version

```diff
@@ -1 +0,0 @@
-v1.1.1-0461be9*v1.1*1*1*1*1*0461be9
```

Comment:
It's important that this file does not get deleted since it is responsible for providing version information in deployments that do not have access to Git

Reply:
I didn't delete it. At least not on intention. It seems this file changes automatically, do I need to commit it? Currently I leave it out of my commits.

Reply:
Ok - it looks like the file is being removed via this commit. We should probably add some documentation to reduce confusion in the future, but the best thing is probably to either commit your changes to the file if they increase the patch version number, or just leave it alone. It's a best effort to provide other users of this code with version information that is up to date.